src/main/webapp/assets/js/tests/indexScript.test.js

```diff
@@ -0,0 +1,78 @@
+// ESLint doesn't know that our functions are defined in other files.
+/* eslint-disable no-undef */
+
+/**
+ * Test suite for all functions that control the distribution of map markers.
+ */
+describe('Distribution of Map Markers', function() {
+  /**
+   * Tests for getRepresentativeCollegeSample().
+   */
+  describe('#getRepresentativeCollegeSample()', function() {
+    it('should return only one location if two are too close to each other', function() {
+      const collegeLocations = [
+        {
+          LAT: 23,
+          LON: 23,
+        },
+        {
+          LAT: 23.2,
+          LON: 22.8,
+        },
+      ];
+      expect(getRepresentativeCollegeSample(collegeLocations).length).to.be.equal(1);
+    });
+
+    it('should return all locations if they are far enough apart from each other', function() {
+      const collegeLocations = [
+        {
+          LAT: 0,
```

Comment:
Since you have integars that are the same in multiple places, would it be better to include them as consts and have the "too close" locations be that const+0.5?

Reply:
That's a good idea! I'll make those changes now :)